tests/generic_views/test_base.py

```diff
@@ -286,6 +287,14 @@ def test_direct_instantiation(self):
         response = view.dispatch(self.rf.head('/'))
         self.assertEqual(response.status_code, 405)
 
+    def test_no_self_in_as_view_signature(self):
+        """
+        Test to check that as_view function's signature does not contain self.
+        """
+        view = SimpleView.as_view()
+        signature = inspect.signature(view)
+        self.assertNotIn("self", signature.parameters.keys())
```

Comment:
Single quotes please, and you also don't need `.keys()` to check key containment on mappings:
```suggestion
        self.assertNotIn('self', signature.parameters)
```

Reply:
Sure, my bad.